### ISSUE TITLE
Crash with Windows UI scaling and window positions 'center' and 'mouse'

### ISSUE THREAD

**Issue by sysrage** [nwjs/nw.js#7414]
NWJS Version : 0.44.4
Operating System : Windows 10 x64

### Expected behavior

The window should appear centered in the screen, when `position: 'center'` is used. The window should appear at the mouse cursor, when `position: 'mouse'` is used.

### Actual behavior

When a package.json manifest contains `position: 'center'`, the application never starts when NW.js 0.44.4 is being used. Additionally, if the manifest contains `position: 'mouse'`, the window shows in the top-left of the screen (not where the mouse is).

Downgrading NW.js back to 0.44.3 immediately resolves both issues. Running NW.js 0.44.4 with `--disable-features=nw2` also resolves the issues.

### How to reproduce

```
nw.Window.open(startUrl, { position: 'center' }, (mainWindow) => {});
```
OR
```
nw.Window.open(startUrl, { position: 'mouse' }, (mainWindow) => {});
```

**Comment by sysrage** [comment]
This issue should block issue #7230.

**Comment by gpetrov** [comment]
Maybe this is related to the crashes we are experiencing as well: see https://github.com/nwjs/nw.js/issues/7412

**Comment by rogerwang** [comment]
@sysrage I tried the code by opening nw and enter it in the devtools. It works for me.

Please provide a sample so I can reproduce the crash, thanks.

**Comment by sysrage** [comment]
There might be something strange going on, as the following code has *not* failed once or twice in my testing. However, it usually does fail (tested on two computers). Also, passing the `--disable-features=nw2` parameter is no longer fixing the problem. Once it starts failing, it continues to fail until I remove `position: 'center'` from main.js.

**package.json:**
```json
{
    "name": "nw-fail",
    "main": "./main.js",
    "homepage": "."
}
```

**main.js**
```js
nw.Window.open('./index.html', { position: 'center' }, (mainWindow) => {});
```

**index.html**
```
Any HTML Here.
```

**Comment by sysrage** [comment]
I believe this issue is related to Windows UI scaling. If I have "Change the size of text, apps, and other items" set to 175% in Display Settings, NW.js crashes with `position: 'center'`. If I set scaling to 100%, the window opens just fine.

**Comment by sysrage** [comment]
This behavior appears to be resolved with the 0.44.5 test build mentioned in Issue #7412 (https://dl.nwjs.io/live-build/nw44/20200316-123559/b37f3d15b/v0.44.5/).

**Comment by rogerwang** [comment]
Close as the fix is released in 0.44.5
